engine/src/main/java/org/terasology/rendering/cameras/OpenVRStereoCamera.java

```diff
@@ -15,23 +15,27 @@
  */
 package org.terasology.rendering.cameras;
 
+import org.terasology.rendering.openvrprovider.OpenVRProvider;
 import org.lwjgl.opengl.GL11;
 import org.terasology.math.MatrixUtils;
 import org.terasology.math.geom.Matrix4f;
 import org.terasology.math.geom.Vector3f;
 import org.terasology.registry.CoreRegistry;
-import org.terasology.rendering.oculusVr.OculusVrHelper;
 import org.terasology.rendering.world.WorldRenderer;
 import org.terasology.rendering.world.WorldRenderer.RenderingStage;
 
 import static org.lwjgl.opengl.GL11.GL_PROJECTION;
 import static org.lwjgl.opengl.GL11.glMatrixMode;
 
 /**
- * Camera which can be used to render stereoscopic images of the scene for the Oculus Rift.
+ * Camera which can be used to render stereoscopic images of the scene for VR.
  *
  */
-public class OculusStereoCamera extends Camera {
+public class OpenVRStereoCamera extends Camera {
+    private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(OpenVRStereoCamera.class);
```

Comment:
You could shorten this a bit with a couple imports, dunno why the extra qualification snuck in :-)

`private static final Logger logger = LoggerFactory.getLogger(OpenVRStereoCamera.class);`